fixtures/link.fixture.js

```diff
@@ -0,0 +1,35 @@
+import Link from "@khanacademy/wonder-blocks-link";
+
+import {describe, expect, test, render, simulate, sleep} from "./helpers.js";
+
+function randomLink() {
+    return "#" + String(Math.random()).slice(2);
+}
+
+describe("Link", () => {
+    test("primary", async () => {
+        const element = await render(
+            <Link href={randomLink()}>Hello, world!</Link>,
```

Comment:
Random URLs are used to prevent subsequent test runs from interfering with each other.